packages/calypso-e2e/src/lib/pages/gutenberg-editor-page.ts

```diff
@@ -427,7 +427,7 @@ export class GutenbergEditorPage {
 		const frame = await this.getEditorFrame();
 
 		await Promise.all( [
-			this.page.waitForNavigation( { waitUntil: 'networkidle', url: url } ),
```

Comment:
This bit [was flaky in my testing](https://teamcity.a8c.com/buildConfiguration/calypso_WPComTests_coblocks_Playwright_desktop/7168493?buildTab=tests&expandedTest=build%3A%28id%3A7168493%29%2Cid%3A2709). It doesn't seem to be needed anymore (as we handle refresh retries), so it should be safe to remove.